apps/s_time.c

```diff
@@ -240,16 +224,10 @@ int s_time_main(int argc, char **argv)
                                    www_path);
             if (buf_len <= 0 || SSL_write(scon, buf, buf_len) <= 0)
                 goto end;
-            while ((i = SSL_read(scon, buf, sizeof(buf))) > 0 ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_READ ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_WRITE)
```

Comment:
Hmm...

I tried this but it did not help:
```
diff --git a/apps/s_socket.c b/apps/s_socket.c
index 804ab5b..5b0e141 100644
--- a/apps/s_socket.c
+++ b/apps/s_socket.c
@@ -179,7 +179,8 @@ int do_server(int *accept_sock, const char *host, const char
     asock = BIO_socket(BIO_ADDRINFO_family(res), BIO_ADDRINFO_socktype(res),
                        BIO_ADDRINFO_protocol(res), 0);
     if (asock == INVALID_SOCKET
-        || !BIO_listen(asock, BIO_ADDRINFO_address(res), BIO_SOCK_REUSEADDR)) {
+        || !BIO_listen(asock, BIO_ADDRINFO_address(res),
+                       BIO_SOCK_REUSEADDR | BIO_SOCK_NODELAY)) {
         BIO_ADDRINFO_free(res);
         ERR_print_errors(bio_err);
         if (asock != INVALID_SOCKET)
```

still tls1.2 -www -reuse
28410 connections in 13.43s; 2115.41 connections/user sec, bytes read 104177885
28410 connections in 31 real seconds, 3666 bytes read per connection

and tls1.3 -www -reuse
8615 connections in 15.03s; 573.19 connections/user sec, bytes read 38285060
8615 connections in 31 real seconds, 4444 bytes read per connection